browser_history/generic.py

```diff
@@ -166,22 +169,46 @@ def history(self, history_paths=None, sort=True, desc=False):
                                    .strptime(d, '%Y-%m-%d %H:%M:%S')
                                    .replace(tzinfo=_local_tz), url)
                                   for d, url in cursor.fetchall()]
-                histories.extend(date_histories)
+                output_object.entries.extend(date_histories)
                 conn.close()
         if sort:
-            histories.sort(reverse=desc)
-        return histories
+            output_object.entries.sort(reverse=desc)
+        return output_object
+
+
+class Outputs():
+    """
+    A generic class to implement encapsulation & provide easy implementation
+    when choosing output in JSON, CSV or other formats.
```

Comment:
This could be worded better. Maybe something like this?
```suggestion
    A generic class to encapsulate history outputs and to
    easily convert them to JSON, CSV or other formats.
```